bears/general/InvalidLinkBear.py

```diff
@@ -33,14 +33,20 @@ def find_links_in_file(file):
             match = regex.search(line)
             if match:
                 link = match.group()
-                code = InvalidLinkBear.get_status_code_or_error(link)
+                code = InvalidLinkBear.get_status_code_or_error(link, timeout)
                 yield line_number + 1, link, code
 
-    def run(self, filename, file):
+    def run(self, filename, file, timeout: str):
```

Comment:
Exactly :)
Sounds good

On Wed, Mar 16, 2016 at 6:02 PM, Ibrahim Jarif <notifications@github.com>
wrote:

> In bears/general/InvalidLinkBear.py
> <https://github.com/coala-analyzer/coala-bears/pull/241#discussion_r56324954>
> :
>
> >                  yield line_number + 1, link, code
> >
> > -    def run(self, filename, file):
> > +    def run(self, filename, file, timeout: str):
>
> Alright. I didn't understand that at first. I'll change the data type to
> int and directly set timeout: int=2 in the param.
> Correct?
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/coala-analyzer/coala-bears/pull/241/files/efe45fed5ff826e1624388bd89842e9d2d508229#r56324954>
>